letsencrypt/client/network2.py

```diff
@@ -534,6 +530,8 @@ def fetch_chain(self, certr):
         """
         if certr.cert_chain_uri is not None:
             return self._get_cert(certr.cert_chain_uri)[1]
+        else:
```

Comment:
https://www.python.org/dev/peps/pep-0008/: `Be consistent in return statements. Either all return statements in a function should return an expression, or none of them should. If any return statement returns an expression, any return statements where no value is returned should explicitly state this as return None , and an explicit return statement should be present at the end of the function (if reachable).` Did we do otherwise?